src/main/scala/mesosphere/marathon/api/JettyRequestLog.scala

```diff
@@ -0,0 +1,19 @@
+package mesosphere.marathon
+package api
+
+import org.eclipse.jetty.server.NCSARequestLog
+import org.slf4j.LoggerFactory
+
+class JettyRequestLog extends NCSARequestLog {
+
+  val lineSepLength = System.lineSeparator().length
+
+  private[this] val log = LoggerFactory.getLogger(getClass.getName)
```

Comment:
What about using `StrictLogging`?

Reply:
This code was copied from Chaos. Will do!

Reply:
copied code from Chaos. Will update.

Reply:
✅